### PR TITLE
Upgrade to Scalafmt 1.1.0

### PR DESCRIPTION
**Pull request by densh**

### CODE REVIEW

**Comment by densh** [comment]
@olafurpg

```
[warn] Error in /Users/denys/.src/native/nir/src/main/scala/scala/scalanative/nir/Attrs.scala: <input>:34: error: identifier expected but : found
final case class Attrs(inline: Inline = MayInline,
                             ^
```
I suspect that's due to inline being prematurely considered a keyword, which it is not at the moment.

File: benchmarks/src/main/scala/cd/Motion.scala [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package cd
 
-class Motion(val callsign: CallSign,
-             val posOne: Vector3D,
-             val posTwo: Vector3D) {
+class Motion(val callsign: CallSign, val posOne: Vector3D, val posTwo: Vector3D) {
```
Comment (densh) [inline benchmarks/src/main/scala/cd/Motion.scala]: @olafurpg 82 column line with 80 column limit.

File: benchmarks/src/main/scala/tracer/Engine.scala [hunk 2]
```diff
@@ -52,8 +52,7 @@ class Engine(val config: EngineConfiguration) {
   }
 
   // 'canvasContext' can be null if raytracer runs as benchmark
-  def renderScene(scene: Scene,
-                  canvasContext: CanvasRenderingContext2D): Unit = {
+  def renderScene(scene: Scene, canvasContext: CanvasRenderingContext2D): Unit = {
```
Comment (densh) [inline benchmarks/src/main/scala/tracer/Engine.scala]: @olafurpg 82 column line with 80 column limit.

File: javalib/src/main/scala/java/io/PrintStream.scala [hunk 3]
```diff
@@ -213,8 +213,7 @@ class PrintStream private (_out: OutputStream,
     }
   }
 
-  @inline private[this] def ensureOpenAndTrapIOExceptions(
-      body: => Unit): Unit = {
+  @inline private[this] def ensureOpenAndTrapIOExceptions(body: => Unit): Unit = {
```
Comment (densh) [inline javalib/src/main/scala/java/io/PrintStream.scala]: @olafurpg 82 column line with 80 column limit.

File: javalib/src/main/scala/java/io/PrintWriter.scala [hunk 4]
```diff
@@ -145,8 +145,7 @@ class PrintWriter(protected[io] var out: Writer, autoFlush: Boolean)
     }
   }
 
-  @inline private[this] def ensureOpenAndTrapIOExceptions(
-      body: => Unit): Unit = {
+  @inline private[this] def ensureOpenAndTrapIOExceptions(body: => Unit): Unit = {
```
Comment (densh) [inline javalib/src/main/scala/java/io/PrintWriter.scala]: @olafurpg 82 column line with 80 column limit.

File: javalib/src/main/scala/java/net/URI.scala [hunk 5]
```diff
@@ -746,8 +745,7 @@ final class URI private () extends Comparable[URI] with Serializable {
    * occur in pairs as above
    */
 
-  private def equalsHexCaseInsensitive(first: String,
-                                       second: String): Boolean = {
+  private def equalsHexCaseInsensitive(first: String, second: String): Boolean = {
```
Comment (densh) [inline javalib/src/main/scala/java/net/URI.scala]: @olafurpg 82 column line with 80 column limit.

**Comment by densh** [comment]
@olafurpg It looks like there is a regression wrt maxColumn handling.

**Comment by olafurpg** [comment]
The maxColumn change is intentional, there was an off-by-one error causing scalafmt to break too eagerly by one character. You can get the previous behavior by bumping the maxColumn down by one.  Scalafmt does not guarantee to alway fit exactly within the provided limit, in these 82 char cases scalafmt deems this OK since the closing `)` fits within 80. That's how it's always worked, for better or worse.

**Comment by densh** [comment]
@olafurpg This looks like a consistent off by 2 now.

**Comment by olafurpg** [comment]
The only change in the line wrapping implementation from before is that a single `<` comparison became a `<=`. 

The inline change is indeed caused by it being considered a keyword by default. Can you try?

```
runner.dialect = Scala212
```

**Comment by olafurpg** [comment]
The only change in the line wrapping implementation from before is that a single `<` comparison became a `<=`. 

The inline change is indeed caused by it being considered a keyword by default. Can you try?

```
runner.dialect = Scala212
```

**Comment by densh** [comment]
Here is the output with `runner.dialect = Scala212`:

```
Reformatting...
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t391.scala: <input>:2: error: identifier expected but def found
def fun1(def x: Int): Int = x; // the "def x" is illegal
           ^
Reformatting... (0.43 %, 35 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/t1785.scala: <input>:6: error: ; expected but [ found
(new t1785)[Int](1)
             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/t1803.scala: <input>:2: error: : expected but ) found
class B extends A { override def foo[A](b) = b }
                                         ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t4069.scala: <input>:4: error: malformed xml literal
<div>
            ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t4091.scala: <input>:1: error: expected start of definition
private a
        ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t4163.scala: <input>:4: error: <- expected but = found
x = 3
        ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t4217.scala: <input>:2: error: case expected but } found
42 match { }
             ^
Reformatting... (2.07 %, 170 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t4584.scala: <input>:1: error: error in unicode escape
class A { val \u2
                 ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5357.scala: <input>:5: error: => expected but : found
case A: N => 1
          ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5361.scala: <input>:2: error: is not a valid refinement declaration
val x : { val self = this } = new { self => }
            ^
Reformatting... (5.39 %, 442 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5510.scala: <input>:2: error: unclosed string interpolation
val s1 = s"xxx
             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/t3368.scala: <input>:4: error: ; expected but xml start found
def x = <![CDATA[hi & bye]]> <![CDATA[red & black]]>
                               ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5702-neg-bad-and-wild.scala: <input>:10: error: illegal start of simple pattern
case List(1, _*,) => // bad use of _* (a sequence pattern must be the last pattern)
                      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5702-neg-bad-brace.scala: <input>:14: error: ) expected but } found
case List(1, _*} =>
                     ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5702-neg-bad-xbrace.scala: <input>:17: error: malformed xml literal
<b>{
      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5702-neg-ugly-xbrace.scala: <input>:11: error: malformed xml literal
val <top>{a, z@_*)</top> = xml
        ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t585.scala: <input>:1: error: unclosed comment
/*
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t5856.scala: <input>:10: error: `$$', `$'ident, `$'this, `$'BlockExpr`$'_
val s9 = s"$"
             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t6227.scala: <input>:2: error: classes cannot be implicit
implicit case class IntOps( i: Int ) {
  ^
Reformatting... (9.19 %, 754 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t6597.scala: <input>:3: error: classes cannot be implicit
implicit case class Quux(value: Int) extends AnyVal with T
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t6728.scala: <input>:4: error: ( expected but } found
}
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t7330.scala: <input>:4: error: pattern must be a value
0 match { case Y[_] => }
                      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t7385.scala: <input>:2: error: ( expected but identifier found
do { println("bippy") } while i<10
                                ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t7473.scala: <input>:6: error: <- expected but = found
(for (x = Option(i); if x == j) yield 42) toList
          ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t7715.scala: <input>:8: error: error in interpolated string: identifier, `this' or block expected
days map s"On the $_th day of Christmas" foreach println
                     ^
Reformatting... (14.02 %, 1150 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t7757b.scala: <input>:3: error: expected start of definition

^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8015-ffb.scala: <input>:9: error: ; expected but \f found
def v = y()\u000c()     // was Int does not take parameters
                  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8182.scala: <input>:4: error: illegal start of simple pattern
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8207.scala: <input>:1: error: . expected but } found
class C { import C.this.toString }
                                 ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8217-local-alias-requires-rhs.scala: <input>:3: error: is not a valid block statement
type A
    ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8325-b.scala: <input>:3: error: ) expected but } found
def k(is: Int*} = ???
                ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t8630.scala: <input>:1: error: ; expected but object found
package bobsdelights  abstract class Fruit(  val name: String,  val color: String  )  object Fruits {  object Apple extends Fruit("apple", "red")  object Orange extends Fruit("orange", "orange")  object Pear extends Fruit("pear", "yellowish")  val menu = List(Apple, Orange, Pear)  }
                                                                                      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t877.scala: <input>:3: error: Invalid literal number
trait Foo extends A(22A, Bug!) {}
                    ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t882.scala: <input>:2: error: ] expected but <% found
trait SortedSet[A <% Ordered[A]] {
                  ^
Reformatting... (19.54 %, 1603 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/unicode-unterminated-quote.scala: <input>:2: error: unclosed string literal
val x = \u0022
               ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmlcorner.scala: <input>:2: error: illegal start of simple expression
val wrong = <:bla/>
                  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated1.scala: <input>:2: error: malformed xml literal
val stuff = <a>
              ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated2.scala: <input>:2: error: malformed xml literal
val stuff = <![CDATA[aoeu
              ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated3.scala: <input>:2: error: malformed xml literal
val stuff = <a unclosedattr="aaaa
              ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated4.scala: <input>:2: error: malformed xml literal
val stuff = <!-- comment starts but never ends...
              ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated5.scala: <input>:3: error: malformed xml literal
case <a>
         ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/xmltruncated6.scala: <input>:2: error: malformed xml literal
val stuff = <a>{ "no closing brace"
              ^
Reformatting... (24.20 %, 1985 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/macro-false-deprecation-warning/Impls_Macros_1.scala: <input>:5: error: illegal start of simple expression
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/t8146a.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 5
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/macro-invalidusage-presuper/Macros_Test_2.scala: <input>:3: error: not a valid early definition
class D extends { def x = macro impl } with AnyRef
                  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/build/dbuild-meta-json-gen.scala: <input>:9: error: expected class or object definition
val meta =
^
Reformatting... (29.55 %, 2424 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/existental-slow-compile2.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 4
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/existential-slow-compile1.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 4
Reformatting... (32.88 %, 2697 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/compiler/scala/reflect/quasiquotes/Quasiquotes.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 17
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/pos/largecasetest.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 2
Reformatting... (37.85 %, 3105 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/compiler/scala/reflect/reify/utils/Extractors.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 115
Reformatting... (40.83 %, 3349 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/positions/New1.scala: <input>:3: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/positions/Unsupported2.scala: <input>:3: error: <- expected but = found
for(i = 0 until (xs.size)) {
          ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/completion-implicit-chained/src/Completions.scala: <input>:12: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/ide-bug-1000349/src/CompletionOnEmptyArgMethod.scala: <input>:3: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/ide-bug-1000531/src/CrashOnLoad.scala: <input>:7: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/infix-completion/src/Snippet.scala: <input>:1: error: identifier expected but } found
object Snippet{val x = 123; 1 + 1./*!*/}
                                       ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/infix-completion2/src/Snippet.scala: <input>:1: error: identifier expected but } found
object Snippet{val x = 123; 1 + x./*!*/}
                                       ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/parse-invariants/src/a/A.scala: <input>:128: error: ; expected but with found
trait Barable with Fooable[Barable]
                ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/t1207/src/Completions.scala: <input>:6: error: { expected but } found
}
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/t4287c/src/Foo.scala: <input>:8: error: : expected but ) found
def this(a) = this()
           ^
Reformatting... (46.74 %, 3834 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/t8459/src/IncompleteDynamicSelect.scala: <input>:13: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/presentation/visibility/src/Completions.scala: <input>:23: error: identifier expected but } found
}
    ^
Reformatting... (49.88 %, 4092 / 8203)

Reformatting... (52.33 %, 4293 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/reflect/scala/reflect/internal/ReificationSupport.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 881
Reformatting... (53.57 %, 4394 / 8203)

[warn] Error in /Users/denys/.src/native/nir/src/main/scala/scala/scalanative/nir/Attrs.scala: <input>:34: error: identifier expected but : found
final case class Attrs(inline: Inline = MayInline,
                             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/script/error-messages.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/script/dashi/a.scala: <input>:1: error: expected class or object definition
def setBippy = sys.props("test.bippy") = "dingus"
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/script/t2365/runner.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
Reformatting... (55.94 %, 4589 / 8203)

Reformatting... (58.17 %, 4772 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/compiler/scala/tools/nsc/ast/parser/Scanners.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 624
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/scaladoc/scalacheck/CommentFactoryTest.scala: <input>:54: error: expected start of definition
inline == dst
                 ^
[warn] Error in /Users/denys/.src/native/nscplugin/src/main/scala/scala/scalanative/nscplugin/NirCodeGen.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 1812
Reformatting... (60.54 %, 4966 / 8203)

Reformatting... (62.53 %, 5129 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/script/fact.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/script/second.scala: <input>:3: error: unclosed character literal
scala -nocompdaemon -e 'println("My second argument is " + args(1))'  arg1 arg2
                                                                   ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/script/t1015.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/script/t1017.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled-windows/script/utf8.scala: <input>:1: error: expected class or object definition
#!/bin/sh
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled-windows/script/loadAndExecute/loadAndExecute.scala: <input>:3: error: unclosed character literal
scala -nocompdaemon -i lAndE1.scala lAndE2.scala -e 'println(5: List[String])'
                                                                             ^
Reformatting... (64.53 %, 5293 / 8203)

Reformatting... (65.71 %, 5390 / 8203)

Reformatting... (67.87 %, 5567 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/neg/t2080.scala: <input>:12: error: illegal combination of modifiers: override and abstract for: trait T
override trait T {
 ^
Reformatting... (70.33 %, 5769 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/run/idempotency-partial-functions.scala: <input>:28: error: expected class or object definition
Test.main(null)
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/ambiguous-float-dots2.scala: <input>:3: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/badtok-1.scala: <input>:2: error: unclosed character literal
'42'
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/badtok-2.scala: <input>:3: error: unclosed quoted identifier
`x
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/badtok-3.scala: <input>:2: error: malformed xml literal
<x
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/run/signals.scala: <input>:3: error: expected class or object definition
val manager = scala.tools.util.SignalManager
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/pending/run/t1980.scala: <input>:15: error: Invalid literal number
c: C = C@96d484
         ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/dotless-targs.scala: <input>:2: error: type application is not allowed for postfix operators
def f1 = "f1" isInstanceOf[String] // not ok
                                              ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/for-comprehension-old.scala: <input>:4: error: val in for comprehension must be followed by assignment
for (val x <- 1 to 5 ; y = x) yield x+y       // fail
             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/illegal-stmt-start.scala: <input>:3: error: illegal start of statement
private def bar {}
    ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/lazyvals.scala: <input>:6: error: lazy values may not be abstract
lazy val t: Int
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/literals.scala: <input>:8: error: Non-zero integral values may not have a leading zero.
def leadingZeros: Int  = { 01 }        // line 6: no leading zero
                             ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/macro-deprecate-idents.scala: <input>:2: error: illegal start of simple pattern
val macro = ???
      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/scaladoc/scala/tools/nsc/doc/base/CommentFactoryBase.scala: <input>:705: error: expected start of definition
inline(check(stop))
                ^
Reformatting... (76.40 %, 6267 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/sealed-final-neg.scala: <input>:41: error: expected class or object definition
"Due to SI-6142 this emits no warnings, so we'll just break it until that's fixed."
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t0351.scala: <input>:2: error: by name type not allowed here
def identity[T](x : => T) : (=> T)
                               ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t0503.scala: <input>:1: error: expected class or object definition
val x = new { } with { }
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t1009.scala: <input>:2: error: empty quoted identifier
def `` = "fish"
      ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t1106.scala: <input>:2: error: expected class or object definition
val p = new Par[String]
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t1838.scala: <input>:5: error: `sealed' modifier can be used only for classes
sealed def f = 0
  ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t2275a.scala: <input>:3: error: malformed xml literal
<br>
    ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t2275b.scala: <input>:2: error: malformed xml literal
{<br>}xx
   ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t2968.scala: <input>:27: error: } expected but end of file found

^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t2968b.scala: <input>:8: error: } expected but end of file found

^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t2973.scala: <input>:1: error: ; expected but package found
package foo {} package bar {}
               ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t3189.scala: <input>:2: error: bad simple pattern: use _* to match a sequence
val Array(a,b*) = ("": Any)
                ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t3209.scala: <input>:2: error: expected start of definition
package test
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t3614.scala: <input>:2: error: is not a valid refinement declaration
def v = new ({ def a=0 })
                 ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/neg/t3769.scala: <input>:2: error: malformed xml literal
val x = <b> <c><a></c> {"text"} </b>
          ^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/files/run/literals.scala: <input>:11: error: Invalid literal number
def 1\u03c1\u03b5\u03c4\u03b7 = "alpha rho epsilon tau eta"
      ^
Reformatting... (81.02 %, 6646 / 8203)

Reformatting... (82.35 %, 6755 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/src/scaladoc/scala/tools/nsc/doc/model/diagram/DiagramFactory.scala: org.scalafmt.Error$SearchStateExploded: Search state exploded around line 242
Reformatting... (83.82 %, 6876 / 8203)

Reformatting... (86.58 %, 7102 / 8203)

Reformatting... (90.48 %, 7422 / 8203)

Reformatting... (91.17 %, 7479 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/presentation/akka/src/akka/util/Duration.scala: <input>:283: error: ) expected but integer constant found
def printHMS = "%02d:%02d:%06.3f".format(toHours, toMinutes % 60, toMillis / 1000. % 60)
                                                                                       ^
Reformatting... (94.94 %, 7788 / 8203)

[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/presentation/ide-bug-1000508/src/Foo.scala: <input>:3: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/presentation/ide-bug-1000545/src/CompletionFails.scala: <input>:25: error: identifier expected but } found
}
^
[warn] Error in /Users/denys/.src/native/scalalib/target/scalaSources/2.11.11/test/disabled/presentation/ide-t1000620/src/b/B.scala: <input>:7: error: identifier expected but } found
}
  ^
Reformatting... (97.14 %, 7968 / 8203)

Reformatting... (98.17 %, 8053 / 8203)

Reformatting... (98.27 %, 8061 / 8203)

Reformatting... (98.46 %, 8077 / 8203)

Reformatting... (98.54 %, 8083 / 8203)

Reformatting... (98.61 %, 8089 / 8203)

Reformatting... (98.70 %, 8096 / 8203)

Reformatting... (98.71 %, 8097 / 8203)

Reformatting... (99.10 %, 8129 / 8203)

Reformatting... (100.00 %, 8203 / 8203)
```

And that's quite suprising given that `scalalib/target/scalaSources/` is not tracked by git and `project.git = true` is set. This does not happen with the default dialect. 

`runner.dialect = Scala211` does not have this problem, and doesn't complain on the `inline` being a keyword.

**Comment by olafurpg** [comment]
Scala212 may not be supported by the configuration yet so it fell back to the default config (no .scalafmt.conf), this is a bug I will fix before the next release https://github.com/scalameta/scalafmt/issues/1012

**Pull request closed without merging**